modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,69 +21,94 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
+
+You must specify at least one failure domain for your {product-title} cluster, so that {product-title} can evenly spread virtual machine nodes across domains or groups on a vCenter server. Consider specifying multiple failure domains if you need to provision virtual machine nodes in different datacenters, clusters, datastores, and so on. 
+
+The following example shows a VMware vSphere cluster configuration file with multiple data centers defined for a VMware vCenter. 
 
 [source,yaml]
 ----
 apiVersion: v1
-baseDomain: example.com <1>
-compute: <2>
-- hyperthreading: Enabled <3>
+baseDomain: example.com
+compute:
+- hyperthreading: Enabled
   name: worker
   replicas: 3
   platform:
-    vsphere: <4>
+    vsphere:
       cpus: 2
       coresPerSocket: 2
       memoryMB: 8192
       osDisk:
         diskSizeGB: 120
-controlPlane: <2>
-  hyperthreading: Enabled <3>
+controlPlane:
+  hyperthreading: Enabled
   name: master
   replicas: 3
   platform:
-    vsphere: <4>
+    vsphere:
       cpus: 4
       coresPerSocket: 2
       memoryMB: 16384
       osDisk:
         diskSizeGB: 120
 metadata:
-  name: cluster <5>
+  name: cluster
 ifdef::network[]
 networking:
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   machineNetwork:
   - cidr: 10.0.0.0/16
-  networkType: OVNKubernetes <9>
+  networkType: OVNKubernetes
   serviceNetwork:
   - 172.30.0.0/16
 endif::network[]
 platform:
   vsphere:
-    vcenter: your.vcenter.server
-    username: username
-    password: password
-    datacenter: datacenter
-    defaultDatastore: datastore
-    folder: folder
-    resourcePool: resource_pool <6>
-    diskType: thin <7>
-    network: VM_Network
-    cluster: vsphere_cluster_name <8>
-    apiVIPs: 
-      - api_vip
-    ingressVIPs: 
-      - ingress_vip
+    vcenters:
+    - server: <dns_ip_address>
+      port: 443
+      username: <username>
+      password: <password>
+      datacenters:
+      - <datacenter1_name>
+      - <datacenter2_name>
+    failureDomains:
+    - name: <failure_domain1_name>
+      region: <region1_name>
+      zone: 1
+      server: <dns_ip_address>
+      topology:
+        datacenter: datacenter1
+        computeCluster: "/datacenter1/host/cluster1"
+        networks:
+        - <VM_Network1_name>
+        datastore: "/datacenter1/datastore/datastore1"
+        resourcePool: "/<datacenter1>/host/<cluster1>/Resources/<resourcePool1>"
+        folder: "/<datacenter1>/vm/<folder1>"
+    - name: <failure_domain_2_name>
+      region: <region_2_name>
+      zone: 2
+      server: <dns_ip_address>
+      topology:
+        datacenter: datacenter2
+        computeCluster: "/datacenter2/host/cluster2"
+        networks:
+        - <VM_Network2_name>
+        datastore: "/datacenter2/datastore/test-datastore2"
+        resourcePool: "/<datacenter2>/host/<cluster2>/Resources/<resourcePool2>"
+        folder: "/<datacenter2>/vm/<folder2>"
+        apiVIPs: 
```

Comment:
Need to move the VIPs under the `vsphere` level

Reply:
VIPs now align with `vsphere` level. I also added in `zones` to `compute` and `controlPlane` levels.